cmd/go-qcow2reader-example/read.go

```diff
@@ -29,7 +29,7 @@ func cmdRead(args []string) error {
 		flag.PrintDefaults()
 	}
 	fs.BoolVar(&debug, "debug", false, "enable printing debug messages")
-	fs.IntVar(&bufferSize, "buffer-size", 65536, "buffer size")
+	fs.IntVar(&bufferSize, "buffer-size", 512*1024, "buffer size")
```

Comment:
This is a separate topic and should be another commit

Reply:
Separated to another commit.